src/cli/commands/test/mod.rs

```diff
@@ -161,13 +161,18 @@ impl Display for TestOutput {
 
 fn setup_hint_processor() -> BuiltinHintProcessor {
 	let skip_hint = HintFunc(Box::new(hints::skip));
+	let mock_call_felt_hint = HintFunc(Box::new(hints::mock_call_felt));
 	let mock_call_hint = HintFunc(Box::new(hints::mock_call));
 	let expect_revert_hint = HintFunc(Box::new(expect_revert));
 	let mut hint_processor = BuiltinHintProcessor::new_empty();
 	hint_processor.add_hint(String::from("skip()"), skip_hint);
 	hint_processor.add_hint(String::from("expect_revert()"), expect_revert_hint);
 	hint_processor.add_hint(
-		String::from("mock_call(func_to_mock, mock_ret_value)"),
+		String::from("mock_call_felt(func_to_mock, mock_value)"),
+		mock_call_felt_hint,
+	);
+	hint_processor.add_hint(
+		String::from("mock_call(func_to_mock, mock_value)"),
```

Comment:
I'm wondering if `mock_value` should not be renamed into something like `mock_value_ptr` to better understand the difference between `mock_call_felt` and `mock_call`

Reply:
rn we have:
- for single felts -> mock_call_felt
- for arrays -> mock_call

since mocked values are right above the hint invocation, one will essentially writes:
```
let func_to_mock = some_func_that_returns_a_ptr;
let mock_value = alloc();
// fill this array
{% mock_call(func_to_mock, mock_value) }%
```

For pointers, so it's supposed to be "obvious" that the mock_value is a pointer

Reply:
Ok got it.